src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,101 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
-
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
-
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+import React, { useContext } from 'react';
+
+import { TodoContext } from '../../context/ContextTodo';
+
+import { ETodoStatus, Todo } from '../../types/Todo';
+
+import '../../utils/custom-string-extensions';
+import styles from './TodoFilter.module.scss';
+
+interface ITodosQuery {
+  inputField?: string;
+  filteredBy?: ETodoStatus;
+}
+
+export function getPreparedTodos(todos: Todo[], {
```

Comment:
Why this function is inside `TodoFilter` component
Better move it to separate file as a helper function